lib/context.js

```diff
@@ -105,24 +106,19 @@ api.process = ({activeCtx, localCtx, options}) => {
     if('@base' in ctx) {
       let base = ctx['@base'];
 
-      // clear base
       if(base === null) {
-        base = null;
-      } else if(!_isString(base)) {
-        throw new JsonLdError(
-          'Invalid JSON-LD syntax; the value of "@base" in a ' +
-          '@context must be a string or null.',
-          'jsonld.SyntaxError', {code: 'invalid base IRI', context: ctx});
-      } else if(base !== '' && !_isAbsoluteIri(base)) {
+        // no action
+      } else if(_isAbsoluteIri(base)) {
+        base = parseUrl(base);
+      } else if(_isRelativeIri(base)) {
+        base = parseUrl(prependBase(activeCtx['@base'].href, base));
+      } else {
```

Comment:
This seems right. One thing I tried on my implementation was the following:

```json
{
  "@context": [{
    "@base": "http://foo.bar/./baz/"
  }, {
    "@base": "example/"
  }],
  "@id": "foobar",
  "http:example.org/": "v"
}
```

This should result in:

```json
[
  {
    "@id": "http://foo.bar/baz/example/foobar",
    "http:example.org/": [
      {
        "@value": "v"
      }
    ]
  }
]
```

Reply:
I added this example to test suite.

Reply:
In this example, shouldn't the second context `@base` simply override the first's `@base`? The combination of them in the output seemed surprising to me. What am I missing?

Reply:
It's building the base as it processes things.  You think the secondary one here should override the doc base instead?  So the id would be "{docbase}/example/foobar"?  I could see it going either way.  What crazy person would be doing this anyway?

Reply:
That's how every other `@context` rule works, from what I remember. There's no such thing as partially or progressively updating a definition in a `@context`, rather, you replace it entirely.

Reply:
It seems to me that @base is always resolved against the current base.